### DISCUSSION TITLE
no updates are displayed on the intermediate site

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1897] [Q&A]
Cood you tell me why the intermediate site is not showing updates. I made changes to the 4-1-1 (en. uk) and 4-1-2 (uk) meta files. The view does not show the changes https://ukraine-sdg.github.io/site/4-1-1/. There are no errors in these files.

**Comment by brockfanning** [comment]
@ukraine-sdg Can you please try the fix described in #1884? This fix is needed on the site repository.
